streams/test-utils/src/main/java/org/apache/kafka/streams/processor/api/MockProcessorContext.java

```diff
@@ -533,4 +544,237 @@ public Map<String, Object> appConfigsWithPrefix(final String prefix) {
             }
         };
     }
+
+
+    public InternalProcessorContext getInternalProcessorContext() {
```

Comment:
`MockProcessorContext` is public API and we cannot just add this. -- I assume this was necessary to fix some tests? For this case, we should do it differently.